bin/setup

```diff
@@ -1,39 +1,31 @@
-#!/bin/bash 
+#!/bin/bash
+
 set -e
 
 # Fixes UTF-8 warning ("Your terminal is not configured to receive UTF-8 encoded text.") on some machines.
 export PYTHONIOENCODING="utf-8"
 grep -Fq 'export PYTHONIOENCODING' ~/.bashrc ||
 echo 'export PYTHONIOENCODING="utf-8"' >> ~/.bashrc
 
-# cd to ~/accounting on login
+# Go to ~/accounting on login & activate the virtual environment.
 if [[ $USER == 'ubuntu' ]]; then
     mkdir -p ~/accounting
     cd ~/accounting
-    grep -Fq 'cd ~/accounting' ~/.bashrc || echo 'cd ~/accounting' >> ~/.bashrc
+    grep -Fq 'cd ~/accounting && pipenv shell' ~/.bashrc || echo 'cd ~/accounting && pipenv shell' >> ~/.bashrc
 fi
 
-# Need python-3.6 repository
+# Need python-3.6 repository.
 echo "Setting up python3.6 repository"
 sudo add-apt-repository 'ppa:deadsnakes/ppa'
-sudo apt-get -y update
+sudo apt-get update
 
-# Install all dependencies.
+# Install all system and python dependencies.
```

Comment:
@UmanShahzad "... and activate this project's virtualenv."? (via `pipenv shell` below)

Reply:
@itsjeyd :+1: